cnames_active.js

```diff
@@ -3040,6 +3040,7 @@ var cnames_active = {
   "sum": "ultirequiem.github.io/sum",
   "suman": "devsumanmdn.github.io",
   "summer": "minecraftpublisher.github.io/SummerJS",
+  "sunar": "cname.vercel-dns.com",
```

Comment:
When using Vercel, you'll want the Cloudflare proxy disabled
```suggestion
  "sunar": "cname.vercel-dns.com", // noCF
```